src/common/server.ts

```diff
@@ -83,6 +85,28 @@ async function getRuffVersion(executable: string): Promise<VersionInfo> {
   return { major, minor, patch };
 }
 
+/**
+ * Validate and log executable from given path.
+ */
+async function validateUsingExecutable(executable: string, strategy: string) {
+  try {
+    const ruffVersion = await getRuffVersion(executable);
+    if (!supportsExecutable(ruffVersion)) {
+      var message = `Skip unsupported executable from ${strategy}: ${executable}`;
+      message += `(Reqiuired at least ${versionToString(
+        MINIMUM_SUPPORTED_EXECUTABLE_VERSION,
+      )}, but found ${versionToString(ruffVersion)} instead)`;
```

Comment:
Can we just create a single variable? Like:
```suggestion
      const message = `Skip unsupported executable from ${strategy}: ${executable} (Reqiuired at least ${versionToString(
        MINIMUM_SUPPORTED_EXECUTABLE_VERSION,
      )}, but found ${versionToString(ruffVersion)} instead)`;
```